### PR TITLE
Update project.json schema warnings

### PR DESCRIPTION
**Pull request by troydai**
Including path to the project.json

After fix:
```
$ kpm build
Warning: C:\myk\DNX\src\Microsoft.Framework.Runtime at line 5 - Property code is deprecated. It is replaced by compile
```

### CODE REVIEW

**Comment by msotclas** [comment]
Hi __@troydai__, I'm your friendly neighborhood Microsoft Open Technologies, Inc. Pull Request Bot (You can call me MSOTBOT). Thanks for your contribution!
    <p>
        It looks like you're a Microsoft contributor (Troy Dai). If you're full-time, we DON'T require a Contribution License Agreement. If you are a vendor, please DO sign the electronic Contribution License Agreement. It will take 2 minutes and there's no faxing! https://cla2.msopentech.com.
    </p>

TTYL, MSOTBOT;

**Comment by troydai** [comment]
Ping @davidfowl

**Comment by davidfowl** [comment]
Example from stylecop:

```
c:\users\davifowl\documents\visual studio 14\Projects\WebApplication13\WebApplication13\Class1.cs(8,1): warning : SA1600 : CSharp.Documentation : The class must have a documentation header.
```

**Comment by davidfowl** [comment]
:shipit:

File: src/Microsoft.Framework.Runtime/Project.cs [hunk 1]
```diff
@@ -93,7 +93,7 @@ public static bool HasProjectFile(string path)
             return File.Exists(projectPath);
         }
 
-        public static bool TryGetProject(string path, out Project project, ICollection<FileFormatWarning> warnings = null)
```
Comment (davidfowl) [inline src/Microsoft.Framework.Runtime/Project.cs]: Change these all back.

**Comment by davidfowl** [comment]
nice! :shipit:

**Pull request merged**